lib/plenario_web/controllers/api/aot_controller.ex

```diff
@@ -14,11 +14,14 @@ defmodule PlenarioWeb.Api.AotController do
   plug :check_page
   plug :check_page_size, default_page_size: 500, page_size_limit: 5000
 
-  defp parse_bbox(value) do
+  defp parse_bbox(value, conn) do
     try do
       Poison.decode!(value) |> Geo.JSON.decode()
     rescue
-      _ -> nil
+      _ ->
+        conn
+        |> put_req_header("accept", "application/vnd.api+json")
```

Comment:
We *do not* accept JSON API. I know you're copying in stuff from elsewhere, and that stuff is wrong. I'm opening a bug report for it.